### PR TITLE
docs: Add info on collected data

### PR DESCRIPTION
**Pull request by roman-mazur**
Supervisor data obtained with https://github.com/balena-io/balena-supervisor/pull/1040

Change-type: patch
Signed-off-by: Roman Mazur <roman@balena.io>

### CODE REVIEW

File: pages/learn/more/collected-data.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+title: Collected Data
+---
+# Data collected by balena
```
Comment (xginn8) [inline pages/learn/more/collected-data.md]: Can you update these references to `balena` to `{{ $names.company.lower }}`? There's a templating system that will build the documentation fully.

File: pages/learn/more/collected-data.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+In order to make data driven decisions, balena collects usage related events via a 3rd party analytics service 
+[Mixpanel](https://mixpanel.com/). These events provide us context on the state of application execution steps.
```
Comment (xginn8) [inline pages/learn/more/collected-data.md]: I _think_ reference-style links are preferred, though I don't know that's a hard, general requirement

Comment (roman-mazur) [reply pages/learn/more/collected-data.md]: @xginn8 would you point me to an example?

File: pages/learn/more/collected-data.md (hunk 1 above)
Comment (roman-mazur) [reply pages/learn/more/collected-data.md]: Replaced.

File: pages/learn/more/collected-data.md (hunk 2 above)
Comment (xginn8) [reply pages/learn/more/collected-data.md]: [like this](https://github.com/balena-io/docs/blame/master/pages/learn/getting-started.md#L67)

**Comment by roman-mazur** [comment]
@balena-ci retest please

@xginn8 your comments should be addressed.

File: pages/learn/more/collected-data.md (hunk 2 above)
Comment (roman-mazur) [reply pages/learn/more/collected-data.md]: Thanks, fixed.

**Comment by roman-mazur** [comment]
@balena-ci retest again please - fixed some references

**Pull request merged**